cmd/cloud-controller-manager/README.md

```diff
@@ -0,0 +1,14 @@
+# cloud-controller-manager/example
```

Comment:
side note: this is starting to look like code we should eventually move to https://github.com/kubernetes/cloud-provider-sample in the future